internal/cli/atlas/iam/organizations/invitations/delete_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build unit
+
+package invitations
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/internal/mocks"
+	"github.com/mongodb/mongodb-atlas-cli/internal/test"
+)
+
+func TestDelete_Run(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	mockStore := mocks.NewMockOrganizationInvitationDeleter(ctrl)
+
+	deleteOpts := &DeleteOpts{
+		store: mockStore,
+		DeleteOpts: &cli.DeleteOpts{
+			Entry:   "5a0a1e7e0f2912c554080adc",
+			Confirm: true,
+		},
+		GlobalOpts: cli.GlobalOpts{OrgID: "1"},
+	}
+
+	mockStore.
+		EXPECT().
+		DeleteInvitation(deleteOpts.ConfigOrgID(), deleteOpts.Entry).
+		Return(nil).
+		Times(1)
+
+	if err := deleteOpts.Run(); err != nil {
+		t.Fatalf("Run() unexpected error: %v", err)
+	}
+}
+
+func TestDeleteBuilder(t *testing.T) {
+	test.CmdValidator(
+		t,
+		DeleteBuilder(),
+		0,
+		[]string{flag.Force, flag.OrgID},
+	)
+}
```

Comment:
Could you update the test to check the template as well?

Reply:
If we're recreating/splitting all commands that are shared between Atlas and On-Prem, we want to keep the changes as trivial as possible (providing this commit as an example), so I'd rather not have this as an additional requirement if it can be avoided.

Would you be ok with excluding this change?